### PR TITLE
Fluent mapping Fixes and Addons

### PR DESCRIPTION
**Pull request by jogibear9988**
- Fix Spaces to Tabs
- Fix Association should not Change IsColumn Attribute
- Support Inheritance via Fluent mapping (In Fluent Mapping we have the Problem, that Inherited classes do not have the attributes of the parent class!)

### CODE REVIEW

**Comment by jogibear9988** [comment]
mhmm... test do not work yet... i need to look

**Comment by jjj-herobase** [comment]
* Sorry about the spaces. I've installed an editorconfig extension to avoid that in future contributions. 
* Can an association ever be a column? It's not that I mind either way, I just added it to avoid adding it to every single association property.

**Comment by jogibear9988** [comment]
don't know, but with my new code, when you use association attribute, it does not set iscolumn property, but does also not reset.

File: Source/Mapping/PropertyMappingBuilder.cs [hunk 1]
```diff
@@ -20,8 +20,6 @@ public class PropertyMappingBuilder<T>
 			_entity       = entity;
 			_memberGetter = memberGetter;
 			_memberInfo   = MemberHelper.MemberOf(memberGetter);
-
```
Comment (ili) [inline Source/Mapping/PropertyMappingBuilder.cs]: Why this was removed?

File: Source/Mapping/EntityDescriptor.cs [hunk 2]
```diff
@@ -198,6 +198,12 @@ internal void InitInheritanceMapping()
 
 					var ed = _mappingSchema.GetEntityDescriptor(mapping.Type);
 
+					foreach (var column in this.Columns)
```
Comment (ili) [inline Source/Mapping/EntityDescriptor.cs]: @jogibear9988 what does this for? it looks like this breaks [some tests](https://ci.appveyor.com/project/igor-tkachev/linq2db/build/1.7.6.274/tests) and without it everything is fine...

**Pull request merged**